public/js/util/source-map.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
```

Comment:
This is fine for now to get something started (sourcemaps are preffed off by default right?) but this will leak sourcemaps across navigations (and potentially load wrong sourcemaps if ids are the same across navigations of different sites).

I think it's going to be more trouble than it's worth to try and avoid storing them in the redux state. `loadSourceMap` should be an action creator that does this and checks the state to see if it exists. We gain the ability to faithfully snapshot what's really happening (and see all the sourcemap loading actions), and access the souremaps in tests. We'll have to recreate all of that if we don't store it in redux.

There may be times when we want to avoid keeping the sourcemaps in a state snapshot (because it potentially could be really big), but I think we need to be smarter about hydrating/dehydrating the state anyway.

There are problems with putting it in the state: we'll need to be smart about the hyrdating/dehydrating the `SourceMapConsumer` instance. But as I said above, I think we need a better way handle that anyway (see the transit library).

This is fine for now, not going to block merging this, but just wanted to discuss it.